boms/cloud-oss-bom/pom.xml

```diff
@@ -45,16 +45,16 @@
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <guava.version>30.1.1-android</guava.version>
-    <google.cloud.java.version>0.157.0</google.cloud.java.version>
+    <google.cloud.java.version>0.158.0</google.cloud.java.version>
     <google.cloud.core.version>1.95.4</google.cloud.core.version>
     <io.grpc.version>1.39.0</io.grpc.version>
     <http.version>1.39.2</http.version>
     <protobuf.version>3.17.3</protobuf.version>
     <!-- We don't use gax-bom because it includes the artifacts with 'testlib' classifier.
         When updating gax.version, update gax.httpjson.version too. -->
-    <gax.version>1.66.0</gax.version>
-    <gax.httpjson.version>0.83.0</gax.httpjson.version>
-    <auth.version>0.26.0</auth.version>
+    <gax.version>1.67.0</gax.version>
+    <gax.httpjson.version>0.84.0</gax.httpjson.version>
+    <auth.version>0.27.0</auth.version>
```

Comment:
No gax or google auth library version change for this google-cloud-bom release.

As per https://storage.googleapis.com/java-cloud-bom-dashboard/com.google.cloud/google-cloud-bom/0.158.0/index.html, google-cloud-bom 0.158.0 was built on [the shared dependencies BOM 1.4.0](https://search.maven.org/artifact/com.google.cloud/google-cloud-shared-dependencies/1.4.0/pom) (no change since previous google-cloud-bom release).